### PR TITLE
python@3.8 3.8.10

### PR DESCRIPTION
**Pull request by branchv**
Created with `brew bump-formula-pr`.

`resource` blocks may require updates.

### CODE REVIEW

**Comment by branchvincent** [comment]
i think some patches are no longer needed (will confirm):

```
patching file Lib/distutils/tests/test_build_ext.py
Hunk #3 FAILED at 497.
1 out of 3 hunks FAILED -- saving rejects to file Lib/distutils/tests/test_build_ext.py.rej
```

File: Formula/python@3.8.rb [hunk 1]
```diff
@@ -53,37 +53,20 @@ class PythonAT38 < Formula
...
-
-  # Remove this block when upstream backports arm64 compatibility to the 3.8 branch
-  if Hardware::CPU.arm?
-    # Upstream PRs #20171, #21114, #21224 and #21249
-    patch do
-      url "https://raw.githubusercontent.com/Homebrew/formula-patches/cef51fde/python/3.8.7.patch"
-      sha256 "01ebca9fed99fd0b34abfd198e7d6137e3f851d9d1a07ded837b28c7998e810c"
-    end
-  end
-
```
Comment (branchvincent) [inline Formula/python@3.8.rb:86]: see https://docs.python.org/release/3.8.10/whatsnew/changelog.html#macos (specifically [bpo-41100](https://bugs.python.org/issue41100))

**Comment by branchvincent** [comment]
should i open a separate pr to fix `global`'s test?
```
Error: global: failed
An exception occurred within a child process:
  MethodDeprecatedError: Calling assert_no_match is deprecated! Use refute_match instead.
Please report this issue to the homebrew/core tap (not Homebrew/brew or Homebrew/core), or even better, submit a PR to fix it:
  /usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/global.rb:98
```

**Review by carlocab — APPROVED** [review]
`global` failure on CI is expected. Thanks, @branchvincent!

**Pull request closed without merging**